waspc/data/Cli/templates/new/src/shared/jsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "allowJs": true,
+    "target": "es2018",
+    "baseUrl": ".",
+    "paths": {
+      "*": [
+        "*",
+        "../../.wasp/out/server/node_modules/*",
```

Comment:
Ok, we decided to go with server.
Do we have any idea how to make this "right" in the future? Well, as "right" as possible?

Reply:
I can't think of a solution that doesn't involve separating server, client, and shared npm dependencies.